docs/learn/learn-nomination-pools.md

```diff
@@ -65,6 +65,28 @@ The earnings of the pool are split pro rata to a member's stake in the bonded po
 staking rewards for members will be the same as if they were a nominator). Importantly, slashes are
 also applied proportionally to members who may have been actively bonded.
 
+:::info Nomination Pool Stats
+
+- There can be a maximum of
+  {{ polkadot: <RPC network="polkadot" path="query.nominationPools.maxPoolMembers" defaultValue={65536} /> :polkadot }}
+  {{ kusama: <RPC network="kusama" path="query.nominationPools.maxPoolMembers" defaultValue={65536} /> :kusama }}
+  members (there are currently
+  {{ polkadot: <RPC network="polkadot" path="query.nominationPools.counterForPoolMembers" defaultValue={149} /> :polkadot }}
+  {{ kusama: <RPC network="kusama" path="query.nominationPools.counterForPoolMembers" defaultValue={149} /> :kusama }}
+  members).
+- There can be a maximum of
+  {{ polkadot: <RPC network="polkadot" path="query.nominationPools.maxPools" defaultValue={64} /> :polkadot }}
+  {{ kusama: <RPC network="kusama" path="query.nominationPools.maxPools" defaultValue={64} /> :kusama }}
+  pools (there are currently
+  {{ polkadot: <RPC network="polkadot" path="query.nominationPools.lastPoolId" defaultValue={64} /> :polkadot }}
+  {{ kusama: <RPC network="kusama" path="query.nominationPools.lastPoolId" defaultValue={59} /> :kusama }}
+  pools).
+- {{ polkadot: No limit on :polkadot }}
+  {{ kusama: There can be a maximum of <RPC network="kusama" path="query.nominationPools.maxPoolMembersPerPool" defaultValue={1024} /> :kusama }}
+  members per pool.
```

Comment:
```suggestion
```suggestion
- There can be a maximum of
  {{ polkadot: <RPC network="polkadot" path="query.nominationPools.maxPoolMembers" defaultValue={16384} /> :polkadot }}
  {{ kusama: <RPC network="kusama" path="query.nominationPools.maxPoolMembers" defaultValue={65536} /> :kusama }}
  members (there are currently
  {{ polkadot: <RPC network="polkadot" path="query.nominationPools.counterForPoolMembers" defaultValue={2295} /> :polkadot }}
  {{ kusama: <RPC network="kusama" path="query.nominationPools.counterForPoolMembers" defaultValue={389} /> :kusama }}
  members).
- There can be a maximum of
  {{ polkadot: <RPC network="polkadot" path="query.nominationPools.maxPools" defaultValue={64} /> :polkadot }}
  {{ kusama: <RPC network="kusama" path="query.nominationPools.maxPools" defaultValue={256} /> :kusama }}
  pools (there are currently
  {{ polkadot: <RPC network="polkadot" path="query.nominationPools.lastPoolId" defaultValue={64} /> :polkadot }}
  {{ kusama: <RPC network="kusama" path="query.nominationPools.lastPoolId" defaultValue={115} /> :kusama }}
  pools).
- {{ polkadot: No limit on :polkadot }}
  {{ kusama: There can be a maximum of <RPC network="kusama" path="query.nominationPools.maxPoolMembersPerPool" defaultValue={1024} /> :kusama }}
  members per pool.
```
```